aten/src/ATen/native/Loss.cpp

```diff
@@ -25,18 +25,26 @@ namespace {
 namespace at {
 namespace meta {
 
-TORCH_META_FUNC(smooth_l1_loss)
-(const Tensor& input, const Tensor& target, const int64_t reduction, double beta) {
-  TORCH_CHECK(beta >= 0, "smooth_l1_loss does not support negative values for beta.")
+void meta_func_loss(impl::MetaBase& meta, const Tensor& input, const Tensor& target, int64_t reduction) {
```

Comment:
I don't see anything obviously wrong in the code, but it looks like something about this refactor is causing problems (https://github.com/pytorch/pytorch/runs/5059123196?check_suite_focus=true).

Judging from the error message, maybe the output is still an undefined tensor by the time you call `meta.maybe_get_output().resize_({})`? You could maybe try gdb'ing in to see whether or not the output is properly defined at that point?